src/chk/chk_engine.c

```diff
@@ -901,30 +988,30 @@ chk_engine_query_one(void *args)
 
 	cqxa = streams[tid].st_arg;
 	target = &cqxa->cqxa_target;
-	rc = ds_mgmt_pool_shard_exist(cqxa->cqxa_uuid,  &path);
+	rc = ds_mgmt_tgt_pool_exist(cqxa->cqxa_uuid,  &path);
 	/* We allow the target nonexist. */
```

Comment:
Just curious, if the VOS pool does not exist? does it still need vos_pool_open()/query?

Reply:
If the pool shard does not exist, then no need to open/query the pool.